test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -147,12 +149,108 @@ public SignatureTrustAndValidityVerificationProviderTests(SigningTestFixture fix
             }
         }
 
+        private static FileInfo CreatePackageFile(TestDirectory testDirectory)
```

Comment:
Consider adding this to `SimpleTestPackageContext`. Similar to [`SimpleTestPackageContext.CreateAsStream`](https://github.com/NuGet/NuGet.Client/blob/dev/test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestPackageContext.cs#L109)